docs/Features/Comparison/Hypotenuse.md

```diff
@@ -2,6 +2,8 @@
 sidebar_position: 30
 ---
 
+_Calculates the square root of the sum of squares of two points between two images._
```

Comment:
Maybe use the term "euclidean distance" because the phrasing here is quiet heavy.

Also read the hypothenuse page it deserves more explanations:

> However, for the resulting image the bit depth and number of channels depends on the input options.

It's not clear how this works. Do you chose channels and get a grayscale image? Or is the processing done on each channel independently?